src/rebuild.ts

```diff
@@ -154,9 +154,7 @@ function buildNode(
             node.appendChild(child);
             continue;
           }
-          if (tagName === 'iframe' && name === 'src') {
-            continue;
-          }
+          
```

Comment:
This looks like it might have some un-intended consequences. Any chance you could add tests for this?

Reply:
I think this code is redundant. Even now this code is never executed because `src` attribute is deleted during serialization in `src/snapshot.ts`